README.md

```diff
@@ -0,0 +1,62 @@
+A client library for authenticating with the ISIC Archive from a SPA (single page application).
+
+![npm](https://img.shields.io/npm/v/@isic/client)
+
+
+## Description
+
+The ISIC client provides support for authenticating with the ISIC Archive using the Auth Code with PKCE flow of OAuth2.0.
+
+## Usage
+
+```bash
+$ yarn add @isic/client
+```
+
+```js
+// create an isic client
+const client = new IsicClient(
+  'v1odYySCetBht6DT9svQdAkvmVXrRHOwIIGNk6JG',
+);
+```
+
+```js
+// handle the client signing in
+client.redirectToLogin();
```

Comment:
Perhaps make it more clear that this should be called in response to some event for login, like a button click.